aspnetcore/tutorials/first-mvc-app-xplat/new-field.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Adding a New Field | Microsoft Docs
+author: rick-anderson
+description: 
+keywords: ASP.NET Core,
+ms.author: riande
+manager: wpickett
+ms.date: 04/14/2017
+ms.topic: article
+ms.assetid: 1638bacf-fe7b-4b41-84b0-06a1574b7734
+ms.technology: aspnet
+ms.prod: asp.net-core
+uid: tutorials/first-mvc-app-xplat/new-field
+---
+# Adding a New Field
+
+By [Rick Anderson](https://twitter.com/RickAndMSFT)
+
+When you use EF Code First to automatically create a database, Code First adds a table to the database to help track whether the schema of the database is in sync with the model classes it was generated from. If they aren't in sync, EF throws an exception. This makes it easier to find inconsistent database/code issues.
+
+This tutorial will add a new field to the `Movies` table. One option is to use [EF Code First Migrations](http://docs.efproject.net/en/latest/platforms/aspnetcore/new-db.html) to add a new field to the model and migrate that change to the database - but SQLlite does not support many migration schema operations, so only very simply migrations are possible. See [SQLite Limitations](https://docs.microsoft.com/ef/core/providers/sqlite/limitations) for more information.
+
+For this tutorial we'll drop our database and create a new one when we change the schema. This workflow works well early in development when we don't have any production data to perserve.
+
+See [Migrations limitations workaround](https://docs.microsoft.com/ef/core/providers/sqlite/limitations?branch=Rick-Anderson-patch-3#migrations-limitations-workaround) for one approch to using migrations with SQLite.
+
+## Adding a Rating Property to the Movie Model
```

Comment:
The doc now has only a single h2.  You could add an h2 with a title like "Add the new property to the database"  right before the discussion of choices for updating the database.

The Seed data update is out of place in that case, but I'd move it to right after the model update anyway.